pkg/tagger/tagger.go

```diff
@@ -219,6 +222,12 @@ func (t *Tagger) Tag(entity string, highCard bool) ([]string, error) {
 	if entity == "" {
 		return nil, errors.New("empty entity ID")
 	}
+
+	// Temporary full cardinality mode override
+	if highCard == false && t.fullCard == true {
```

Comment:
nitpick: we don't need the first condition. Or the if actually. `highCard = highCard || t.fullCard` should do the trick